### PR TITLE
docs: Add documentation about hot fixing.

### PR DESCRIPTION
**Pull request by Crash--**

### CODE REVIEW

File: packages/cozy-app-publish/README.md [hunk 1]
```diff
@@ -208,3 +208,21 @@ We apply the semver convention:
...
+If you need to qucikly fix a bug in production, then you have to:
+- fetch the latest branch release 
+- create a new release branch from there (since this is a bug fix, this is a patch version)
+- fix the bug 
+- Make a beta
+- Publish
+- Merge it on master
+
+Exemple: You have to hot fix Drive. The currection `stable` version of Drive is : 1.45.0. Master is on 1.47.0. 
+Then you need to fetch release/1.45.0
```
Comment (Ldoppea) [inline packages/cozy-app-publish/README.md:223]: ⚠️ since there is no blank line between those lines, everything will be collapsed into one single line.

**Review by Ldoppea — APPROVED** [review]

File: packages/cozy-app-publish/README.md [hunk 2]
```diff
@@ -208,3 +208,21 @@ We apply the semver convention:
...
+
+If you need to qucikly fix a bug in production, then you have to:
+- fetch the latest branch release 
+- create a new release branch from there (since this is a bug fix, this is a patch version)
+- fix the bug 
+- Make a beta
+- Publish
+- Merge it on master
+
+Exemple: You have to hot fix Drive. The currection `stable` version of Drive is : 1.45.0. Master is on 1.47.0.
```
Comment (paultranvan) [inline packages/cozy-app-publish/README.md:222]: s/currection/current ?

File: packages/cozy-app-publish/README.md [hunk 3]
```diff
@@ -208,3 +208,21 @@ We apply the semver convention:
 - If the release brings only bug fixes, then we should create a patch version.
 - If the release brings at least one feature, then we should create a minor version.
 - If the release changes its application's permissions, we should create a minor version even if this release is for bug fixes only.
+
+### Hot Fix
+
+If you need to qucikly fix a bug in production, then you have to:
```
Comment (paultranvan) [inline packages/cozy-app-publish/README.md:214]: s/qucikly/quickly

File: packages/cozy-app-publish/README.md [hunk 4]
```diff
@@ -208,3 +208,21 @@ We apply the semver convention:
...
+- fetch the latest branch release 
+- create a new release branch from there (since this is a bug fix, this is a patch version)
+- fix the bug 
+- Make a beta
```
Comment (paultranvan) [inline packages/cozy-app-publish/README.md:218]: Why is it necessary to do a beta for a hot fix?

Comment (Crash--) [reply packages/cozy-app-publish/README.md:218]: To be sure that we didn't break anything when fixing the bug?

Comment (paultranvan) [reply packages/cozy-app-publish/README.md:218]: I would have assumed we already test everything before pushing :)

**Review by paultranvan — APPROVED** [review]

**Pull request merged**